spec/controllers/mailings_controller_spec.rb

```diff
@@ -42,15 +41,15 @@
     end
 
     context 'with user logged in' do
-      let(:user) { FactoryGirl.create(:student) }
```

Comment:
Feel free to go ahead on it. I'm on a dirty branch right now and will stay there for some time as it seems ✌️